ProjectSettings/TagManager.asset

```diff
@@ -16,10 +16,10 @@ TagManager:
   - Bullet
   - FirstPersonWeapon
   - TriggerArea
-  - 
-  - 
-  - 
-  - 
+  - Player
+  - AffectsPlayer
```

Comment:
Why do we need this `AffectsPlayer` layer? as opposed to just fixing the logic in Deathplane?

Reply:
I guess we don't strictly need it? Both approaches feel like they have their pros and cons. The benefit in my opinion is that if we want to extend this behavior to something else (ie. the health pickups) its just a matter of changing a prefab's layer

Reply:
Also good to  remember, somebody else might come in and try to implement picking up, but it won't be obvious to them why it's not working until they either figure out it's something in the layer vs their logic.

Reply:
Yeah I'm open to changing it if you prefer, up to you

Reply:
yeah can u change it?

Reply:
I think after thinking about this further it might be better to keep the layer approach - string comparisons just seem so much messier and though I get its minimal, I could see the performance benefit from using Unity's built in physics helping later down the line. In addition if we decide death planes shouldn't be visible that becomes a super small fix